src/components/grid-list/grid-list.scss

```diff
@@ -64,4 +75,32 @@ md-grid-tile {
   md-grid-tile-footer {
     bottom: 0;
   }
+
+  [md-line] {
+    display: block;
```

Comment:
Is there some stuff here common with `list` that can be refactored into a scss mixin?